concepts/consensus/pos/index.md

```diff
@@ -25,7 +25,12 @@ In each consensus instance a consensus round of three voting steps is executed a
 
 If the committee member receives _2f+1_ Precommits, then by protocol the block proposal is decided and:
 
-- The block is broadcast to the network. The block proposer broadcasts the new block, its [block header](/concepts/system-model/#block-header) containing the block proposal and precommit message seals and an array of the committee members that voted for the block. Other committee members broadcast a (new) block announcement.
-- The block is committed to the ledger. Participants that receive the new block check validity by verifying the seals. Participants that receive an announcement sync to retrieve the block, verifying  by computing block state as it appends the block to the chain.
+- The block is broadcast to the network. The block proposer broadcasts the new block. Other committee members broadcast a (new) block announcement.
```

Comment:
by `the new block` does it mean here a block for a new height? If so I would rephrase as:

```
The decided block is broadcast to the network. The block proposer for the following height broadcasts a new proposal. 
```

If not, then there is probably an error that we need to iron out here